src/ServiceControl.Monitoring.AcceptanceTests/Tests/ApiIntegrationTest.cs

```diff
@@ -42,7 +42,7 @@ string GetString(string url)
             return httpClient.GetStringAsync(url).GetAwaiter().GetResult();
         }
 
-        string MonitoredEndpointsUrl = "http://localhost:1234/monitored-endpoints";
+        string MonitoredEndpointsUrl = "http://localhost:1234/monitored-endpoints?history=1";
```

Comment:
Why is this needed? Is this no longer the default? I'm not seeing this change in the PR.

Reply:
Side note: I think that this PR is not focused enough :/. I've added changes to tests to make them run faster and be able to run only subset easily. 

The reason why I've added the `?history=1` is that the  current aggregation interval is not reported so for integration tests it means we have to wait until current interval has passed to get any data when doing queries. The shorter history period the shorter interval -> I've switched to history period of 1 min.